time_tracker_api/users/users_namespace.py

```diff
@@ -59,3 +72,16 @@ class User(Resource):
     def get(self, id):
         """Get an user"""
         return {}
+
+    @ns.doc('update_user')
+    @ns.expect(user_input_fields)
+    @ns.response(
+        HTTPStatus.BAD_REQUEST, 'Invalid format or structure of the user'
+    )
+    @ns.marshal_with(user_response_fields)
```

Comment:
I think we need to put this into a separate route. I'd like to have something like
```
/users/{user-id}/roles
```
Then, we can have all the HTTP verbs here like:
```
POST: adds a new roles to the user
DELETE: removes an existing role
...
```